services/ui-src/src/components/reports/DrawerReportPage.tsx

```diff
@@ -262,12 +272,13 @@ export const DrawerReportPage = ({ route, validateOnRender }: Props) => {
       )}
       <Box>
         {isReportingOnStandards ? (
-          // table of standards
           <Box>
-            {addStandardsButton}
-            {/* standards table */}
-            <Box></Box>
-            {addStandardsButton}
+            {addStandardsButton()}
```

Comment:
Should we make addStandardsButton a react component instead of a function that returns JSX?

Reply:
I don't really have a strong opinion on this